terraform/environments/analytical-platform-compute/lakeformation-registrations.tf

```diff
@@ -0,0 +1,16 @@
+module "replicated_cadet_bucket" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+  #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
+  source = "github.com/ministryofjustice/terraform-aws-analytical-platform-lakeformation?ref=0.5.0"
+  data_locations = [{
+    data_location = module.mojap_compute_logs_bucket_eu_west_1.s3_bucket_arn
```

Comment:
I think this should be `module.mojap_derived_tables_replication_bucket.s3_bucket_arn`, it's currently pointing to the logging bucket (unless that is intended)